www-client/pybugz/pybugz-0.13-r3.ebuild

```
@@ -0,0 +1,32 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+PYTHON_REQ_USE="readline(+),ssl(+)"
+
+if [ "${PV}" = "9999" ]; then
```

we prefer `[[` over `[` in gentoo:

```suggestion
if [[ "${PV}" == "9999" ]]; then
```

---

You are correct in both cases, so do as is correct.
I was so mistaken about `==` - thanks for pointing me at it!